lib/motsu/src/lib.rs

```diff
@@ -48,3 +48,118 @@ pub mod prelude;
 mod shims;
 
 pub use motsu_proc::test;
+
+#[cfg(all(test))]
+mod tests {
+    #![deny(rustdoc::broken_intra_doc_links)]
+    extern crate alloc;
+
+    use alloy_primitives::uint;
+    use stylus_sdk::{
+        alloy_primitives::{Address, U256},
+        call::Call,
+        prelude::{public, sol_storage, StorageType, TopLevelStorage},
+    };
+
+    use crate::context::{Account, Contract, TestRouter};
+
+    /// Message sender that mocks `msg::sender()` in tests.
+    #[must_use]
+    pub fn msg_sender() -> Address {
```

Comment:
Why is the helper redefined here when it's already defined with the same code in [context.rs](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/423/files#diff-c9edad7c85ffd5b6ce47be67e85ec0460548f9c7b988b60a6ae7874c93f87567R7)? Can't we just import it?

Reply:
This one for test cases. It shows the main disadvantage of an approach, that user should declare that mock by himself. 
Will be removed once stylus sdk will be ready not to cache the `msg::sender()` value